### PR TITLE
feat: add shopLogoUrls to shop settings to allow logo to be rendered from url

### PR DESCRIPTION
**Pull request by kieckhafer**
Impact: **minor**  
Type: **feature**

Adds a field to allow operators to link to an existing logo URL, and use that logo / url as their primary shop logo.

## Breaking changes
None. The URL logo will take precedence, if available, but will not affect anything if it is not added.

## Testing
1. Start the app, see that the default reaction logo is used as the shop logo.
1. Upload a logo via the upload image feature, and see that the uploaded image takes over as the shop logo.
1. Add an image URL,  and see that the image form the URL takes over as the shop logo.
1. Remove the URL, and see the image goes back to the uploaded image.
1. Remove uploaded image, and see it goes back to the default
1. Add a URL (without uploading an image), and see that the URL logo becomes the shop logo.

### CODE REVIEW

File: imports/collections/schemas/shops.js [hunk 1]
```diff
@@ -204,6 +204,22 @@ export const ParcelSize = new SimpleSchema({
...
+ * @property {String} styles optional
+ */
+export const ShopLogoUrls = new SimpleSchema({
+  primaryShopLogoUrl: {
+    type: String,
+    optional: true
+  }
+});
+
+registerSchema("ShopLogoUrls", ShopLogoUrls);
```
Comment (aldeed) [inline imports/collections/schemas/shops.js]: I don't see this being used anywhere. Should the field just be added to shop schema?

Comment (kieckhafer) [reply imports/collections/schemas/shops.js]: Yes.

**Comment by kieckhafer** [comment]
@aldeed schema validation has been added: https://github.com/reactioncommerce/reaction/pull/5227/commits/2d2336bd9ca3273a0d1b24efeea46a8825328756

**Comment by kieckhafer** [comment]
`snyk` has been fixed in another branch, it should be passing once this is merged.

**Comment by willopez** [comment]
@kieckhafer merging develop into this branch should fix the build issues.

**Pull request merged**